models/webhook.go

```diff
@@ -484,6 +504,10 @@ func UpdateHookTask(t *HookTask) error {
 
 // PrepareWebhook adds special webhook to task queue for given payload.
 func PrepareWebhook(w *Webhook, repo *Repository, event HookEventType, p api.Payloader) error {
+	return prepareWebhook(x, w, repo, event, p)
+}
+
+func prepareWebhook(e Engine, w *Webhook, repo *Repository, event HookEventType, p api.Payloader) error {
```

Comment:
Missing `HookEventRepository` on switch case. And missing handle on `GetSlackPayload` and `GetDiscordPayload`.

Reply:
Added missing case to `HookEventRepository`. Will hopefully get change to add handles to `GetSlackPayload` and `GetDiscordPayload` tomorrow.

Reply:
@lunny Done